test-engine/src/main/java/com/aws/jverify/testengine/JVerifyTestEngine.java

```
@@ -196,6 +196,9 @@ public static void verifyFile(SourceFile sourceFile, JVerifyTest annotation, Lis
                 .forEach(dafnyOutput -> {
                     URI source = ((DafnyDiagnostic) dafnyOutput).getSource();
                     var methodIntervals = verificationResultsWithMethodIntervals.sourceFileToMethodIntervals().get(source);
+                    if (methodIntervals == null) {
```

When does this happen? I'm nervous about silently dropping verification results if something went wrong.

---

I don't know. I'll remove it for now.